x/wasm/ibc.go

```diff
@@ -15,12 +15,12 @@ import (
 var _ porttypes.IBCModule = IBCHandler{}
 
 type IBCHandler struct {
-	keeper        Keeper
-	channelKeeper wasmTypes.ChannelKeeper
+	keeper        types.IBCContractKeeper
+	channelKeeper types.ChannelKeeper
 }
 
-func NewIBCHandler(keeper Keeper) IBCHandler {
-	return IBCHandler{keeper: keeper, channelKeeper: keeper.ChannelKeeper}
```

Comment:
Makes sense to add one arg to the constructor to not depend on the internal data structure